ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -532,37 +551,37 @@ export default defineComponent({
       })
     })
 
-    // check if Owner Group that has deceased Owners is valid
-    const isInvalidOwnerGroup = (groupId): boolean => {
-      if ((isTransferToExecutorProbateWill.value || isTransferToExecutorUnder25Will.value) && props.validateTransfer) {
-        const hasRemovedOwners = TransToExec.hasSomeOwnersRemoved(groupId)
-        const hasExecutors = TransToExec.hasAddedExecutorsInGroup(groupId)
-        const hasRemovedAllOwners = TransToExec.hasAllCurrentOwnersRemoved(groupId)
-        const hasValidDocs = TransToExec.hasOwnersWithValidSupportDocs(groupId)
-        const hasOwnersWithoutDeathCert = !TransToExec.isAllGroupOwnersWithDeathCerts(groupId)
+    const isInvalidRegistrationOwnerGroup = (groupId: number) =>
+      hasMixedOwnersInGroup(groupId) && localState.reviewedOwners && !localState.showTableError
 
-        const isInvalid = !(hasRemovedAllOwners && hasValidDocs && hasExecutors && hasOwnersWithoutDeathCert)
+    const isInvalidTransferOwnerGroup = (groupId: number, hasRoleInGroup: boolean) => {
```

Comment:
Please let me know if you have a better name for this function. This one specifically checks the validation of owner groups for TransferToExecutorProbateWill | TransferToExecutorUnder25Will.value | TransferToAdminNoWill

Reply:
`HomeOwnersTable` component is also used in Transfers, so I would avoid using 'registration' word

Reply:
Agreed to leave as is for now